wlauto/core/resolver.py

```diff
@@ -69,7 +69,10 @@ def get(self, resource, strict=True, *args, **kwargs):
                 self.logger.debug('\t{}'.format(result))
                 return result
         if strict:
-            raise ResourceError('{} could not be found'.format(resource))
+            if "version" in kwargs:
+                raise ResourceError('{} (Version:{}) could not be found'.format(resource, kwargs['version']))
```

Comment:
I would make this more generic -- ``kwargs`` basically specify additional criterea for finding the resource. I would port them all.